integration_tests/run_pyspark_from_build.sh

```diff
@@ -333,6 +333,11 @@ EOF
                 --driver-class-path "${PYSP_TEST_spark_driver_extraClassPath}"
                 --conf spark.executor.extraClassPath="${PYSP_TEST_spark_driver_extraClassPath}"
             )
+        elif [[ -n "$PYSP_TEST_spark_jars_packages" ]]; then
```

Comment:
Make `SPARK_SHELL_SMOKE_TEST` support running with `--package` and `--repositories` params